seqseek/chromosome.py

```diff
@@ -61,25 +59,31 @@ def validate_coordinates(self, start, end, loop=False):
 
     @classmethod
     def sorted_chromosome_length_tuples(cls, assembly):
-        chromosome_lengths = cls.ASSEMBLY_CHROMOSOMES[assembly]
-        return sorted(chromosome_lengths.items(),
+        # TODO: simplify
+        name_to_accession = cls.ASSEMBLY_CHROMOSOMES[assembly]
+        chromosome_length_tuples = []
+        for name, accession in name_to_accession.items():
+            if name in name_to_accession and accession in ACCESSION_LENGTHS:
```

Comment:
If name comes from name_to_accession.items() there's no way for "name in name_to_accession" to return false, so safe to omit.